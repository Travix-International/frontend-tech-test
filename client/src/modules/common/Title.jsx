import React from 'react';
import PropTypes from 'prop-types';

export default class Title extends React.Component {
  static propTypes = {
    className: PropTypes.string,
    children: PropTypes.any.isRequired,
  }

  static defaultProps = {
    className: '',
  }


  render() {
    const { children } = this.props;
    return (
      <h5 className={`Title ${this.props.className}`}>
        {children}
      </h5>
    );
  }
}
