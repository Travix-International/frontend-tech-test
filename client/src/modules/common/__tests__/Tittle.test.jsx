import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Title from '../Title';

const text = 'the title';

describe('Common.Tittle', () => {
  it('should render with props', () => {
    const title = shallow(<Title>{text}</Title>);

    expect(toJson(title)).toMatchSnapshot();
  });
});
