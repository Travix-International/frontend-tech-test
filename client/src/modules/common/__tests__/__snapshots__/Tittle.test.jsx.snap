// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Common.Tittle should render with props 1`] = `
<h5
  className="Title "
>
  the title
</h5>
`;
